v1/src/test/java/com/google/cloud/teleport/templates/PubsubToBigQueryLT.java

```diff
@@ -58,7 +58,7 @@ public class PubsubToBigQueryLT extends TemplateLoadTestBase {
           TestProperties.specPath(),
           "gs://dataflow-templates/latest/PubSub_Subscription_to_BigQuery");
   // 35,000,000 messages of the given schema make up approximately 10GB
-  private static final long NUM_MESSAGES = 35000000L;
+  private static final int NUM_MESSAGES = 35000000;
```

Comment:
nitpick: you can replace `35000000` with `35_000_000` to have a more legible constant

```suggestion
  private static final int NUM_MESSAGES = 35_000_000;
```

Reply:
Neat! Thanks!